src/stream-ciphers.rst

```diff
@@ -375,43 +375,43 @@ The server dutifully encrypts that message using the predicted IV
        & = E(k, IV_A \xor G)
    \end{aligned}
 
-That ciphertext, C\ :sub:`M`, is exactly the ciphertext block Alice
-would have had if her plaintext block was G. So, depending on what the
-data is, Mallory has figured out if Alice has a criminal record or not,
-or perhaps some kind of embarrassing disease, or some other issue that
-Alice really expected the server to keep secret.
+The ciphertext, C\ :sub:`M`, is exactly the ciphertext block Alice
+would have if her plaintext block was G. Depending on the
+data, Mallory found out if Alice has a criminal record or not,
+or some kind of embarrassing disease, or some other issue that
+Alice really expects the server to keep secret.
 
-Lessons learned: don't let IVs be predictable. Also, don't roll your own
-cryptosystems. In a secure system, Alice and Mallory's records probably
-wouldn't be encrypted using the same key.
+Lessons learned: do not let IVs be predictable. Also, do not roll your own
+cryptosystems. In a secure system, records of Alice and Mallory probably
+are not encrypted using the same key.
 
 Attacks on CBC mode with the key as the IV
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Many CBC systems set the key as the :term:`initialization vector`. This seems
-like a good idea: you always need a shared secret key already anyway. It
-yields a nice performance benefit, because the sender and the receiver
-don't have to communicate the IV explicitly, they already know the key
-(and therefore the IV) ahead of time. Plus, the key is definitely
-unpredictable because it's secret: if it were predictable, the attacker
-could just predict the key directly and already have won. Conveniently,
-many block ciphers have block sizes that are the same length or less
+Many CBC systems set the key as the :term:`initialization vector`. This seems like
+a good idea as you always need a shared secret key. It
+yields a nice performance benefit because the sender and receiver
+do not have to communicate to the IV explicitly. They already know the key
```

Comment:
"to communicate to the IV" makes no sense; the IV is the object of the communication.

Reply:
Perfect!

Reply:
Updated